.github/CODEOWNERS

```diff
@@ -95,6 +102,8 @@
 /packages/logging-controller/CHANGELOG.md         @MetaMask/confirmations @MetaMask/wallet-framework-engineers
 /packages/message-manager/package.json            @MetaMask/confirmations @MetaMask/wallet-framework-engineers
 /packages/message-manager/CHANGELOG.md            @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/multichain-api-middleware/package.json  @MetaMask/earn @MetaMask/wallet-framework-engineers
+/packages/multichain-api-middleware/CHANGELOG.md  @MetaMask/earn @MetaMask/wallet-framework-engineers
```

Comment:
and here 😅

Reply:
Fixed here: 947b53ed418b2af26a26f2ba7c68d69cb3afba93